internal/parser.go

```diff
@@ -2730,6 +2730,10 @@ func inLiteralIM(p *parser) bool {
 }
 
 func inExpressionIM(p *parser) bool {
+	// if p.oe.contains(a.Table) {
```

Comment:
let's remove these lines if no longer relevant

Reply:
Ah yeah, forgot to remove that after debugging the issue

Reply:
Done in [`7f82f6b`](https://github.com/withastro/compiler/pull/954/commits/7f82f6be742414f534d575074863dcf8082403d3) !